content/docs/user-guide/deploying/heroku.md

```diff
@@ -0,0 +1,167 @@
+# Heroku
+
+[Heroku](https://heroku.com) is the platform suitable for deploy of simple
```

Comment:
`for deploy of` --> `for deployment of`

Reply:
thanks! could you please fix it yourself?